src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    Ball(String color, int number) {
```

Comment:
For the future: [pay attention to access modifiers](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#pay-attention-to-access-modifiers)

Reply:
Okey, thanks)